en/simulation/multi_vehicle_simulation_gazebo.md

```diff
@@ -27,7 +27,7 @@ Tools/gazebo_sitl_multiple_run.sh [-m <model>] [-n <number_of_vehicles>] [-w <wo
 
  - `<target>`: build target, e.g: `px4_sitl_default` (default), `px4_sitl_rtps`
 
-Each vehicle instance is allocated a unique MAVLink system id (1, 2, 3, etc.) and can be accessed from a unique remote offboard UDP port (14540, 14541, 14542, etc.).
+Each vehicle instance is allocated a unique MAVLink system id (1, 2, 3, etc.) and can be accessed from a unique remote offboard UDP port (14540, 14541, 14542, etc.). 
```

Comment:
Pending discussion https://github.com/PX4/PX4-user_guide/issues/1035#issuecomment-777095536 - I think that once you hit 14549 all instances need to connect to this same port.

Reply:
You're right. At 14549 it behaves like 14550 :smile: 
https://github.com/PX4/PX4-Autopilot/blob/176d932f23ac9e8909bfe5711cd65dae2d20119e/ROMFS/px4fmu_common/init.d-posix/rcS#L111